### PR TITLE
[ocf] Add start function and multiple resources for OCF test cases

### PR DESCRIPTION
**Pull request by cuiyanx**
**TOTAL:**  40 of 41 passed

Signed-off-by: Cui Yan <yanx.cui@intel.com>

### CODE REVIEW

File: tests/test-ocfclient.js [hunk 1]
```diff
@@ -66,147 +50,212 @@ var expectedPlatform = {
...
 
 function onError(error) {
     if (error.deviceId) console.log("Device error: " + error.deviceId);
 }
 
-client.on("update", onUpdate);
-client.on("devicefound", onDevicefound);
-client.on("platformfound", onPlatformfound);
-client.on("error", onError);
+var FoundStringFlag = 1;
```
Comment (jimmy-huang) [inline tests/test-ocfclient.js]: is there reason the flags are using 0/1, not boolean like before?

File: tests/test-ocfclient.js [hunk 2]
```diff
@@ -66,147 +50,212 @@ var expectedPlatform = {
...
-client.findResources({ resourceType:"core.test" }, onfound).then(function(resource) {
-    assert(true, "OCFClient: find OCF server resources");
+client.on("devicefound", onDevicefound);
+client.on("platformfound", onPlatformfound);
+client.on("update", onUpdate);
+client.on("error", onError);
+
+ocf.start();
+
+var ResourcesStringFlag = 0;
```
Comment (jimmy-huang) [inline tests/test-ocfclient.js]: same here

File: tests/test-ocfclient.js (hunk 1 above)
Comment (cuiyanx) [inline tests/test-ocfclient.js]: No, there isn't. I have mistake. I will correct this.

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**